docker/binaries/Dockerfile.bn.amd64

```diff
@@ -0,0 +1,33 @@
+FROM debian:stable-slim as prod
```

Comment:
Shall we add a comment at the top indicating the expected purpose of this Dockerfile?

Reply:
I can add a comment there - something like "A Dockerfile to build a distributable container image from pre-existing binaries" ??

Reply:
Yes that sounds great!